### DISCUSSION TITLE
Component render

### DISCUSSION THREAD

**Discussion by Givx** [livewire/livewire#5387] [7. Bugs (archived)]
Hello I have a component laravel classic. 

```PHP
 @props([
    'label',
    'description',
    'fieldNameActive',
    'fieldNameProfil'
])

<div class="w-[30%] p-5 mb-5 rounded border border-gray-200">
    <div class="flex items-center justify-between" 
        x-data="{ {!! $fieldNameActive !!} : @entangle(' {!! $fieldNameActive !!} ').defer }">
```

and i have a bug with the render...

```PHP
<div class="flex items-center justify-between" 
  x-data="{ activeChess : window.Livewire.find('xlivO83oMm39KITrXP68').entangle(' <?php echo $fieldNameActive; ?> ').defer }">
```

the probleme it's <?php echo $fieldNameActive; ?>

Thanks

**Comment by joshhanley** [comment]
@Givx using `@entangle` is already blade evaluating PHP, so you don't need the echo brackets `{!! !!}` or `{{ }}`. So you would write it like this: `@entangle($fieldNameActive)`

It looks like you are building a blade component, ao you can actually make use of `wire:model` if you'd prefer instead of using a prop called "fieldNameActive". See here for details https://laravel-livewire.com/docs/2.x/alpine-js#livewire-directives-from-blade-components

Hope this helps!
